apps/css-workshop/vite.config.mjs

```diff
@@ -2,30 +2,31 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import { defineConfig, type Plugin } from 'vite';
+import { defineConfig } from 'vite';
 import fs from 'fs';
 import { resolve } from 'path';
 import { minify } from 'html-minifier';
 
 export default defineConfig({
   base: './',
+  root: './pages',
   build: {
+    outDir: '../dist',
     rollupOptions: {
       input: Object.fromEntries(
-        getComponentList().map((name) => [name, resolve(__dirname, `./backstop/tests/${name}.html`)]),
+        getComponentList().map((name) => [
+          name,
+          resolve(__dirname, `./pages/${name}.html`),
+        ]),
       ),
-      output: {
-        dir: './backstop/minified',
-        compact: true,
-        assetFileNames: `assets/[name].[ext]`,
-      },
+      output: { compact: true },
```

Comment:
To confirm, the output to `backstop/minified` is not required anymore?

Earlier:
![image](https://github.com/iTwin/iTwinUI/assets/45748283/9a30e23e-c967-4cf4-98d6-be7c2ebb8328)

If so, should all occurrences of `backstop/minified` be removed (e.g. copyrightLinter.js)?

Reply:
done https://github.com/iTwin/iTwinUI/pull/1715/commits/fb1b863ca419cc7a43f41370d52d62518c452389

there is one more in `fieldset.html` but it is just used as page content and not important